src/foam/nanos/dig/DUGRuleAction.js

```diff
@@ -51,10 +51,13 @@ foam.CLASS({
         @Override
         public void execute(X x) {
           PM pm = new PM(getClass(), rule.getDaoKey(), rule.getName());
+          DAO dugDigestConfigDAO = (DAO) x.get("dugDigestConfigDAO");
+          DUGDigestConfig dugDigestConfig = (DUGDigestConfig) dugDigestConfigDAO.find(rule.getSpid());
```

Comment:
So here, if the config is null, create a normal HTTPSink, otherwise the HTTPDigestSink.